eng/pipelines/templates/stages/archetype-net-release.yml

```
@@ -43,6 +43,19 @@ stages:
         dependsOn: Signing
         condition: and(succeeded(), ne(variables['SetDevVersion'], 'true'), ne(variables['Skip.Release'], 'true'), ne(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-net-pr'))
         jobs:
+          - ${{if ne(artifact.skipPublishPackage, 'true')}}:
+            - job: InstallationCheck
```

The yaml indention would be easier to read if you used a condition, but then it would also show up as skipped in the job list, which you may not want.

```suggestion
          - job: InstallationCheck
            condition: eq('${{ artifact.skipPublishPackage }}', 'true')}}:
```